client/src/components/graph/graph.tsx

```diff
@@ -300,12 +285,17 @@ class Graph extends React.Component<GraphProps, GraphState> {
       },
       updateOverlay: false,
       testImageSrc: null,
+      isDeepZoomSourceValid: true,
     };
   }
 
   // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
   componentDidMount() {
-    window.addEventListener("resize", this.handleResize);
+    window.addEventListener(
+      "resize",
+      throttle(this.handleResize, THROTTLE_MS, { trailing: true })
```

Comment:
the handler is now throttled, so openseadragon can resize to the viewport properly. Note that `THROTTLE_MS` is 16ms for an ideal 60fps. Similar to requestAnimationFrame but less guarantee

On the other hand, not sure which version of lodash introduced requestAnimationFrame in its debounce and throttle functions, but it's there if we really want to use it!

https://github.com/lodash/lodash/blob/main/src/debounce.ts

Reply:
Ooh yes yes 🙏 Thanks so much for catching this, @codemonkey800 !! Just pushed a commit to address this 🤩